src/tools/wast2wasm.cc

```diff
@@ -90,8 +93,12 @@ static Option s_options[] = {
     {FLAG_VERBOSE, 'v', "verbose", nullptr, NOPE,
      "use multiple times for more info"},
     {FLAG_HELP, 'h', "help", nullptr, NOPE, "print this help message"},
+    {FLAG_DEBUG_PARSER, 0, "debug-parser", nullptr, NOPE,
+     "Turn on debugging the parser of wast files"},
     {FLAG_DUMP_MODULE, 'd', "dump-module", nullptr, NOPE,
      "print a hexdump of the module to stdout"},
+    {FLAG_EXCEPTIONS, 0, "except", nullptr, NOPE,
```

Comment:
I agree I didn't know the best way to introduce this. Changing to "future-exceptions".